recipes/checkatlas/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "checkatlas" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2bf72a64d383d260dfd33c17f7d823f303973947b339fd32ee39e2a8d54ec241
+
+build:
+  number: 0
+  entry_points:
+    - checkatlas = checkatlas.__main__:main
+    - checkatlas-workflow = checkatlas.checkatlas_workflow:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - llvmlite >=0.39.1,<0.40.0
+    - numba >=0.56.4,<0.57.0
+    - numpy >=1.23.5,<2.0.0
+    - pip
+    - poetry >=1.5.1,<2.0.0
+    - python
```

Comment:
```suggestion
    - python >=3.10
```

Reply:
Please add minimum python version constraints under the host and run sections if applicable. The pypi source states python >=3.10.